tests/boardfarm_plugins/boardfarm_prplmesh/devices/prplmesh_prplwrt.py

```diff
@@ -0,0 +1,69 @@
+###############################################################
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+# SPDX-FileCopyrightText: 2020 the prplMesh contributors (see AUTHORS.md)
+# This code is subject to the terms of the BSD+Patent license.
+# See LICENSE file for more details.
+###############################################################
+
+import boardfarm
+from boardfarm.devices import OpenWrtRouter
+from environment import ALEntityPrplWrt, _get_bridge_interface
+from .prplmesh_base import PrplMeshBase
+from sniffer import Sniffer
+
+
+class PrplMeshPrplWRT(OpenWrtRouter, PrplMeshBase):
+    """prplWRT burned device with prplMesh installed."""
+
+    model = "prplWRT"
+    prompt = ['root\\@OpenWrt:/#', '/#', '@OpenWrt:/#']
+    wan_iface = "eth1"
+    uboot_eth = "eth0_1"
+    agent_entity = None
+    controller_entity = None
+
+    def __init__(self, *args, **kwargs):
+        """Initialize device."""
+        self.args = args
+        self.kwargs = kwargs
+        config = kwargs.get("config", kwargs)
+        self.docker_network = config.get("docker_network",
+                                         "prplMesh-net-{}".format(self.unique_id))
+        self.role = config.get("role", "agent")
+        if self.role == "controller":
+            self.controller_entity = ALEntityPrplWrt(self.name, is_controller=True)
```

Comment:
This can't be right, the device argument is missing... I guess this should be
```suggestion
            self.controller_entity = ALEntityPrplWrt(self.name, self, is_controller=True)
```
In that case, there is no need to pass self.name, right? The __init__ function can just get it from the device argument.

Reply:
Than better to drop `name` argument at all. Fixed in #1358.